app/assess/routes.py

```diff
@@ -93,9 +109,20 @@ def view_application(application_id, fund_id, round_id):
     if not application_data:
         abort(404)
 
+    status_data_ALL = get_status_data(
+        APPLICATION_STORE_API_HOST, application_id
+    )
+    status_NOT_COMPLETED = get_status_not_completed(status_data_ALL)
+    status_COMPLETED = get_status_completed(status_data_ALL)
+    if not status_data_ALL:
+        abort(404)
+
     return render_template(
         "project_summary.html",
         application_data=application_data,
         fund_data=fund_data,
         round_data=round_data,
+        status_data_ALL=status_data_ALL,
+        status_COMPLETED=status_COMPLETED,
```

Comment:
Need to update the names across your application :)

Reply:
Haha, I did then I changed it back thinking you may have meant the functions only. Done now!